django_bmemcached/memcached.py

```diff
@@ -8,7 +10,6 @@ class BMemcached(memcached.BaseMemcachedCache):
     A.K.A BMemcached.
     """
     def __init__(self, server, params):
-        import bmemcached
```

Comment:
I am trying to remember if there was a reason to put it here but I don't think that there is one because If somehow in django you used this, it means that for sure python-binary-memached was installed

Reply:
I got a linter complaint about it (not an error) so tried moving it, and it worked, so kept it at the top?